zeb_dt_test.py

```diff
@@ -0,0 +1,98 @@
+import random
```

Comment:
should we put this file as well under `src/transformers/models/decision_transformers`?